versioned_docs/version-24.04/update/update-centreon-ha.md

```diff
@@ -0,0 +1,189 @@
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```bash
+mv /usr/share/centreon/www/install /var/lib/centreon/installs/install-update-`date +%Y%m%d`
+sudo -u apache /usr/share/centreon/bin/console cache:clear
+```
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```bash
+mv /usr/share/centreon/www/install /var/lib/centreon/installs/install-update-`date +%Y%m%d`
+sudo -u www-data /usr/share/centreon/bin/console cache:clear
+```
+</TabItem>
+</Tabs>
+
+### Removing cron jobs
+
+The RPM upgrade restores the unnecessary cron jobs that were deleted during the installation procedure. Remove them on the two central nodes to avoid concurrent executions:
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
+
+```bash
+rm -f /etc/cron.d/centreon
+rm -f /etc/cron.d/centstorage
+rm -f /etc/cron.d/centreon-auto-disco
+systemctl restart crond
+```
+
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```bash
+rm -f /etc/cron.d/centreon
+rm -f /etc/cron.d/centstorage
+rm -f /etc/cron.d/centreon-auto-disco
+systemctl restart crond
+```
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```bash
+rm -f /etc/cron.d/centreon
+rm -f /etc/cron.d/centstorage
+rm -f /etc/cron.d/centreon-auto-disco
+systemctl restart cron
+```
+</TabItem>
+</Tabs>
+
+### Updating Centreon extensions
+
+The Centreon extensions are also updated *via* the WUI, from the "Administration > Extensions > Manager" menu, by clicking the "Update all" button.
+
+### Updating the Monitoring Connectors
+
+In order to maintain compatibility between the [Monitoring Connectors](https://docs.centreon.com/docs/monitoring/pluginpacks/) and the installed plugins (that have just been updated on the central server), the Monitoring Connectors must also be updated in the WUI from the **Configuration > Monitoring Connector Manager** menu.
+
+### Exporting Engine/Broker configuration
+
+Generate and export new Engine/Broker configuration files for all pollers (central included) in **Configuration > Pollers** with these options checked:
+
+* Generate Configuration Files
+* Run monitoring engine debug (-v)
+* Move Export Files
+
+Then restart them **one at a time** from the same menu and make sure to select the **"Restart" method, not "Reload"**, if the `centreon-engine` and/or `centreon-broker` packages have been updated.
+
+In addition, the `cbd-sql` service must be restarted **on the central master server**:
+
+```bash
+service cbd-sql restart
+```
+
+The `cbd` service must also be restarted on [Centreon Remote Servers](https://docs.centreon.com/docs/23.10/installation/architectures/#distributed-architecture):
+
+```bash
+service cbd restart
+```
+
+## Resuming cluster resource management
+
+Now that the update is finished, the resources can be managed again:
+
+```bash
+pcs property set maintenance-mode=false
+pcs resource cleanup ms_mysql
```

Comment:
Is the cleanup really required?

Reply:
In my opinion it is not required as the update process on a 4-node setup does not touch the database nodes, and therefore there should be no extra constraints on that side because of the update process.

Reply:
Removed